src/components/src/index.ts

```diff
@@ -102,7 +99,10 @@ import SocialNetworkSharing from './SocialNetworkSharing/socialNetworkSharing';
 import CartClear from './CartClear/cartclear';
 import ProductDisplayItemDetails from './ProductDisplayItemDetails/productdisplayitem.details';
 import B2bSideMenu from './B2bSideMenu/b2b.sidemenu';
+import BarcodeScanner from './BarcodeScanner/barcodescanner';
```

Comment:
Do we need this here?

Reply:
This file will be completely removed in future PR's.  Apologies there was no real purpose of that move -- got brought into this PR by accident.  I'll revert to avoid confusion

Reply:
reverted!

Reply:
No worries, I've already approved the PR.